### ISSUE TITLE
install package

### ISSUE THREAD

**Issue by VojislavM** [IRNAS/koruza-openwrt-packages#1]
@kostko Can you please guide me with more details how to install a package on the nodeW.
I can not find the feeds.conf file. All I found was costumfeeds.conf.
When I try to run:

``` bash
./scripts/feeds update koruza
```

in the terminal I got:

``` bash
-ash: ./scripts/feeds: not found
```

**Comment by kostko** [comment]
Where did you run this? You need to run these commands inside an OpenWrt DD (trunk) checkout.

**Comment by VojislavM** [comment]
I got ruter terminal thru ssh and tried to find those files and run commands there...

**Comment by kostko** [comment]
No. To build this, you need an OpenWrt toolchain on your machine:
- [Get the source.](https://dev.openwrt.org/wiki/GetSource) (you need trunk)
- [Usage instructions.](https://wiki.openwrt.org/doc/howto/build)
